docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+     46506407   cluster-name-mgr1              169.##.###.##   10.###.###.##   2     4096     wdc07        managers
+     ...
+
+     Load Balancers
+     ID                                     Name                Address                                          Type
+     ID-string                              cluster-name-mgr    cluster-name-mgr-1234567-wdc07.lb.bluemix.net    mgr
+     ...
+
+     Subnets
+     ID        Gateway          Datacenter
+     ID-number 10.###.###.##   wdc07
+
+     NFS Volumes
+     ID              ID-number
+     Mount Address   fsf-wdc0701b-fz.adn.networklayer.com:/ID_number/data01
+     Datacenter      wdc07
+     Capacity        20
+     Type            ENDURANCE_FILE_STORAGE
+     Tier Level      10_IOPS_PER_GB
+
+     OK
+    ```
+    {% endraw %}
+
+After creating the cluster, you must [log in to Docker UCP and download the Docker UCP client certificate bundle](#use-the-universal-control-plane).
+
+## Use the Universal Control Plane
+Docker EE for IBM Cloud uses [Docker Universal Control Plane (UCP)](/datacenter/ucp/2.2/guides/) to provide integrated container management and security, from development to production.
+
+### Access UCP
+Before you begin, [create a cluster](#create-swarms). Note the UCP password.
+
+1. Retrieve your UCP password by using the cluster **Name** and **ID** that you made when you [created the cluster](#create-swarms).
+   ```bash
+   $ docker logs cluster-name_ID
+
+   ...
+   ucp_password = UCP-password
+   ...
+   ```
+
+2. Retrieve the load balancer IP address.
+   ```bash
+   $ bx d4ic list --sl-user user.name.1234567 --sl-api-key api_key
+   ```
+
+3. Copy the **UCP URL** from the `bx d4ic list` command, and in your browser navigate to it.
+
+4. Log in to UCP. Your credentials are `admin` and the UCP password from the `docker logs` command, or the credentials that your admin created for you.
+
+### Download client certificates
+[Download the client certificate bundle](/datacenter/ucp/2.2/guides/user/access-ucp/cli-based-access/#download-client-certificates) to create objects and deploy services from a local Docker client.
+
+1. [Access UCP](#access-ucp).
+2. Under your user name (for example, *admin*), click *My Profile*.
+3. Click *Client Bundles* > *New Client Bundle*. A zip file is generated.
+4. In the GUI, you are now shown a label and public key. You can edit the label by clicking the pencil icon and giving it a name, e.g., _d4ic-ucp_.
+5. In a terminal, navigate and unzip the client bundle.
+
+   ```bash
+   $ cd Downloads && unzip ucp-bundle-admin.zip
+   ```
+
+   > Keep your client bundle handy
+   >
+   > Move the certificate environment variable directory to a safe and accessible location on your machine. You'll use it a lot!
+
+6. From the client bundle directory, update your `DOCKER_HOST` and `DOCKER_CERT_PATH` environment variables by running the `env.sh` script.
```

Comment:
Nit: You are not really running the script, but loading its contents into your environment.